prime/src/components/borrow/actions/UniswapAddLiquidityActionCard.tsx

```diff
@@ -369,8 +364,14 @@ export default function UniswapAddLiquidityActionCard(props: ActionCardProps) {
     />
   );
 
-  const input0MaxString = isToken0Selected ? maxString0 : maxString1;
-  const input1MaxString = isToken0Selected ? maxString1 : maxString0;
+  const maxes = [accountState.assets.token0Raw, accountState.assets.token1Raw];
+  const max0 = maxes[isToken0Selected ? 0 : 1];
+  const max1 = maxes[isToken0Selected ? 1 : 0];
+  const maxString0 = Math.max(0, max0 - 1e-6).toFixed(6);
+  const maxString1 = Math.max(0, max1 - 1e-6).toFixed(6);
+  // Not to be confused with token0Amount and token1Amount, these are the amounts associated with the inputs
```

Comment:
I know what you mean but this comment is kinda confusing. Maybe just say something like "In the case that token0 is selected, we have to flip flop which amount is first and which is second"